eox_nelp/mfe_config_api/v1/views.py

```diff
@@ -0,0 +1,65 @@
+"""The generic views for MFE_CONFIG API. Nelp flavour."""
+
+import json
+
+from django.conf import settings
+from django.http import JsonResponse
+from eox_theming.configuration import ThemingConfiguration as theming
+from rest_framework import status
+
+from eox_nelp.edxapp_wrapper.mfe_config_view import MFEConfigView
+
+
+class NelpMFEConfigView(MFEConfigView):
+    """
+    Provides an API endpoint to get the MFE_CONFIG from site configuration with
+    nelp extra fields.
+    Based on https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/mfe_config_api/views.py.
+    **Usage**
+    Get common config:
+    GET eox-nelp/api/mfe_config/v1
+    **GET Response Values**
+    ```
+    {
+        "BASE_URL": "https://name_of_mfe.example.com",
+        "LANGUAGE_PREFERENCE_COOKIE_NAME": "example-language-preference",
+        "CREDENTIALS_BASE_URL": "https://credentials.example.com",
+        "DISCOVERY_API_BASE_URL": "https://discovery.example.com",
+        "LMS_BASE_URL": "https://courses.example.com",
+        "LOGIN_URL": "https://courses.example.com/login",
+        "LOGOUT_URL": "https://courses.example.com/logout",
+        "STUDIO_BASE_URL": "https://studio.example.com",
+        "LOGO_URL": "https://courses.example.com/logo.png",
+        "THEME_OPTIONS": {"CONTROL_THEME_OPTIONS": "values"...}
+        "CUSTOM_PRIMARY_COLORS": {"pgn-color-primary-base": "interactive_color"}
+        ... and other django setting or tenant config vars mapped with MFE_CONFIG_MAP.
+    }
```

Comment:
would be great if you add an example of  MFE_CONFIG_MAP , how to set and its result

Reply:
is ok like? 
https://github.com/eduNEXT/eox-nelp/pull/23/commits/f8e3afbb1004014fc879c75cd241ddc8932668a8